src/test/java/com/github/onsdigital/dpjavadurationparse/ParseDurationTest.java

```diff
@@ -0,0 +1,115 @@
+package com.github.onsdigital.dpjavadurationparse;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+import java.time.Duration;
+import java.time.format.DateTimeParseException;
+
+import static com.github.onsdigital.dpjavadurationparse.ParseDuration.parseDuration;
+import static org.junit.jupiter.api.Assertions.*;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Test for {@link ParseDuration}.
+ */
+public class    ParseDurationTest {
+
+    @Test
+    public void durationsWithPT()  {
+        // Given
+        // a set of valid positive durations in strings, check their associated parsed duration object is correct
+
+        // Positive Durations
+        assertEquals(Duration.ofHours(3).plusMinutes(45).plusSeconds(50), parseDuration("PT3h45m50s"));
+        assertEquals(Duration.ofSeconds(0), parseDuration("PT0s"));
```

Comment:
Not a deal breaker but rather than inlining the strings to be passed to `parseDuration(...)` these should really be constants.

`private static final String PARSEDURATIONPT0s = "PT0s";`